html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <section>
+      <h1><strong>Coding Form!</strong></h1>
+      <p id="description">description</p>
+      <p></p>
+      <form id="survey-form">
+        <label id="email-label">Email<input id="email" /></label>
+        <label>why is coding cool?</label>
+        <select id="why-is-coding-cool?">
```

Comment:
Your `label` on line 13 should have a `for` attribute that matches your `id` on line 14